pages/common/dd.md

```diff
@@ -19,6 +19,14 @@
 
 `dd if=/dev/zero of={{file_1GB}} bs=1024 count=1000000`
 
+- Generate a system backup into a IMG file and show the progress:
+
+`dd if=/dev/{{drive_device}} of={{/path/to/file.img}} status=progress`
+
+- Restore a drive from an IMG file and show the progress:
+
+`dd if={{/path/to/file.img}} of=/dev/{{drive_device}} status=progress`
```

Comment:
```suggestion
`dd if={{path/to/file.img}} of=/dev/{{drive_device}} status=progress`
```
As far as I can remember, this doesn't have to be absolute.

Reply:
It doesn't however I would never recommend not. 
This command as you probably know is dangerous. Always use absolute with dd

Reply:
I can't really think of what could to wrong when using the wrong `img` file.

Reply:
Writing the wrong file comes to mind

Reply:
Would that be a problem? You could just rewrite with the right one.

Reply:
Of course, but any command that writes to a file can destroy a file that you don't intend if you specify an incorrect path.

Reply:
Yes however DD is probably one of the easiest commands to mess up on.

I've nearly wiped my entire system before from a mix-up of if and of. 

If we are going to keep it as relative, maybe we should add a small note to say this command is dangerous. 

And although this would apply for a lot of commands, dd is up there with rm /*

Reply:
It's the users's responsibility to ensure that they don't wreck their system — tldr-pages simply provides examples on how to use the command.

Reply:
Ok, well in that case, we'll use relative. Commits applied in 30 seconds.